WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1231,6 +1229,7 @@ def sendFailoverRequest( self, status = '', minorStatus = '' ):
       isValid = RequestValidator().validate( request )
       if not isValid["OK"]:
         self.log.error( "Failover request is not valid", isValid["Message"] )
+        self.__report( minorStatus = 'Failover Request Failed' )
```

Comment:
No it's not missing. This method is not supposed to return anything. I've actually removed the other `return`s